versioned_docs/version-2.0.0/concepts/serialization/structures.md

```diff
@@ -17,53 +17,48 @@ An Account is a structure that represented a user on a legacy Casper network. Al
 ## AddressableEntity {#addressable-entity}
 
 An Addressable Entity is a structure that represents an entity on a Casper network. The addressable entity consists of the following fields:
-
 - [`package_hash`](./types.md#package-hash)
 
 - [`byte_code_hash`](./types.md#byte-code-hash)
 
-- [`entry_points`](./types.md#entrypoints)
-
 - [`protocol_version`](./types.md#protocolversion)
 
-- `main_purse`: The entity's main purse `URef`. You may find information on `URef` serialization [here](./primitives.md#clvalue-uref).
+- [`main_purse`]: The entity's main purse `URef`. You may find information on `URef` serialization [here](./primitives.md#clvalue-uref).
 
 - [`associated_keys`](./types.md#associatedkey)
 
 - [`action_thresholds`](./types.md#entity-action-thresholds)
 
-- [`message_topics`](./types.md#message-topics)
-
 - [`entity_kind`](./types.md#entity-kind)
 
+
 ## Block {#serialization-standard-block}
 
 A block is the core component of the Casper linear blockchain, used in two contexts:
 
 1.  A data structure containing a collection of transactions. Blocks form the primary structure of the blockchain.
 2.  A message that is exchanged between nodes containing the data structure as explained in (1).
 
-Each block has a globally unique ID, achieved by hashing the contents of the block.
-
-Each block points to its parent. An exception is the first block, which has no parent.
-
-A block is structurally defined as follows:
-
--   `hash`: A hash over the header of the block.
--   `header`: The header of the block that contains information about the contents of the block with additional metadata.
--   `body`: The block's body contains the proposer of the block and hashes of deploys and transfers contained within it.
+The block is a polymorphic structure that assumes one of the well-defined variants:
+* `V1` which type is `BlockV1` (it has a binary `prefix` of 0)
+* `V2` which type is `BlockV2` (it has a binary `prefix` of 1)
 
-Further, a block may consist of one of the following types:
+To byte-serialize a `Block` you first need to serialize it's binary prefix and then write the bytes of the byte-serialized representation of it's variant.
 
-- `Version1`: A legacy block created prior to the Condor upgrade.
+## BlockV1 {#block-v1}
 
-- `Version2`: A modern block.
+This type encapsulates block data that used to be produced in versions of the nodes prior to 2.0.
```

Comment:
May can we simplify the sentences like;

This **type**;
- encapsulates block data that used to be produced in versions of the nodes prior to 2.0.
- represents a historical block that was produced prior to 2.0

Each **block**; 
- has a globally unique ID, achieved by hashing the contents of the block.
- points to its parent. An exception is the first block, which has no parent.